src/commands/cacert/export.ts

```diff
@@ -41,23 +39,23 @@ export default class Export extends Command {
 
   async run() {
     const { flags } = this.parse(Export)
-    const ctx: any = {}
+    const kube = new KubeHelper(flags)
     const cheHelper = new CheHelper(flags)
-    const cheTasks = new CheTasks(flags)
-    const apiTasks = new ApiTasks()
-    const tasks = new Listr([], { renderer: 'silent' })
 
-    tasks.add(apiTasks.testApiTasks(flags, this))
-    tasks.add(cheTasks.verifyCheNamespaceExistsTask(flags, this))
+    if (!await kube.hasReadPermissionsForNamespace(flags.chenamespace)) {
+      throw new Error(`E_PERM_DENIED - Permission for Che server namespace "${flags.chenamespace}" are required`)
```

Comment:
`E_PERM_DENIED - Permission denied: no read access to '${flags.chenamespace}' namespace`

Reply:
ok